examples/devices/lighty-actions-device/src/main/java/io/lighty/netconf/device/action/processors/ResetActionProcessor.java

```diff
@@ -47,35 +46,33 @@
 import org.w3c.dom.Node;
 import org.w3c.dom.NodeList;
 
-public class ResetActionProcessor extends ActionServiceDeviceProcessor {
+public final class ResetActionProcessor extends ActionServiceDeviceProcessor {
 
     private static final Logger LOG = LoggerFactory.getLogger(ResetActionProcessor.class);
 
-    private final CurrentAdapterSerializer adapterSerializer;
-    private ActionDefinition actionDefinition;
-    private Absolute actionPath;
     private final Reset resetAction;
+    private Absolute path;
+    private ActionDefinition definition;
+    private final CurrentAdapterSerializer adapterSerializer;
 
-    @SuppressWarnings("rawtypes")
-    public ResetActionProcessor(final Reset resetAction, final BindingDOMCodecServices codecServices) {
+    public ResetActionProcessor(final Reset resetAction, final Absolute path, final ActionDefinition definition,
+            final BindingDOMCodecServices codecServices) {
         this.resetAction = resetAction;
-        final ConstantAdapterContext constantAdapterContext = new ConstantAdapterContext(codecServices);
-        this.adapterSerializer = constantAdapterContext.currentSerializer();
+        this.path = path;
+        this.definition = definition;
+        this.adapterSerializer = new ConstantAdapterContext(codecServices).currentSerializer();
     }
 
     @SuppressWarnings({"rawtypes", "unchecked", "checkstyle:IllegalCatch"})
     @Override
     protected CompletableFuture<Response> execute(final Element requestXmlElement,
-            final Entry<Absolute, ActionDefinition> actionEntry) {
-        this.actionDefinition = actionEntry.getValue();
-        this.actionPath = actionEntry.getKey();
+            final ActionDefinition actionDefinition) {
```

Comment:
Please remove `final ActionDefinition actionDefinition`. It's not used.

Reply:
Done.